client/containers/DashboardSettings/CollectionSettings/CollectionSettings.tsx

```diff
@@ -98,6 +109,21 @@ const CollectionSettings = () => {
 			pubPubIcon: 'pub',
 			sections: [<CommunityOrCollectionLevelPubSettings />],
 		},
+		collection.kind !== 'tag' &&
+			depositTarget.service === 'datacite' && {
```

Comment:
We only show the `DOI` tab on the collection dashboard settings page if the community targets Datacite. Crossref deposits are still handled via the collection metadata editor for Crossref-enabled communities (default).